EPRN.Waste.API/Services/JourneyService.cs

```diff
@@ -95,7 +95,7 @@ public async Task<WasteRecordStatusDto> GetWasteRecordStatus(int journeyId)
             {
                 JourneyId = journey.Id,
                 WasteBalance = journey.Quantity ?? 0,
-                WasteRecordReferenceNumber = journey.ReferenceNumber,
+                WasteRecordReferenceNumber = journey.ReferenceNumber ?? "WR00012",
```

Comment:
Put a comment or a TODO comment explaining why this is hard coded, and that it'll be removed in time

Reply:
Done.